src/Components/Publishing/EditorialFeature/Components/Eoy2018Culture.tsx

```diff
@@ -1,21 +1,306 @@
+    return isHeader
+  }
+
+  makeSectionArrays = () => {
+    const {
+      article: { sections },
+    } = this.props
+    const chapters = []
+    let currentChapter = null
+
+    sections.map((section, i) => {
+      if (this.sectionIsHeader(section)) {
+        if (currentChapter === null) {
+          currentChapter = 0
+        } else {
+          currentChapter += 1
+        }
+      }
+
+      if (i !== 0) {
+        if (chapters.length && chapters[currentChapter]) {
+          chapters[currentChapter].push(section)
+        } else {
+          chapters[currentChapter] = [section]
+        }
+      }
+    })
+    return chapters
+  }
+
+  getSections = sections => {
+    return sections.map((section, i) => {
+      if (this.sectionIsHeader(section)) {
+        return this.sectionHeaderText(section, i)
+      } else {
+        switch (section.type) {
+          case "text": {
+            return this.sectionText(section, i)
+          }
+          case "image_collection": {
+            return this.sectionImageCollection(section, i)
+          }
+          case "image_set": {
+            return this.sectionImageSet(section, i)
+          }
+          default: {
+            return null
+          }
+        }
+      }
+    })
+  }
+
+  render() {
+    const { article } = this.props
+    const { chapters } = this.state
+    const introText =
+      article.sections && article.sections[0] && article.sections[0].body
+
+    return (
+      <ArticleWrapper>
+        <Nav canFix color="black" backgroundColor="white" />
+
+        <ArticleHeader>
+          <ArticleTitle size="16" weight="medium" px={55} pb={10} pt={20}>
+            <div>Year In Visual Culture</div>
+            <div>2018</div>
+          </ArticleTitle>
+          <HeaderImg src={article.hero_section.url} />
+        </ArticleHeader>
+
+        <TextContainer px={[20, 55]} py={30} mx="auto">
+          <Box maxWidth={1400} mx="auto">
+            <Flex
+              pb={20}
+              alignItems="center"
+              flexDirection={["row-reverse", "row"]}
+            >
+              <Box width={["100%", "15%"]}>
+                <Share
+                  url={getArticleFullHref(article.slug)}
+                  title={article.social_title || article.thumbnail_title}
+                />
+              </Box>
+              <Flex width={["100%", "70%"]} maxWidth={1000}>
+                <Author authors={article.authors} />
+                <Date date={article.published_at} />
+              </Flex>
+            </Flex>
+
+            <Box width={["100%", "70%"]} maxWidth={1000} mx="auto">
+              {introText && (
+                <div dangerouslySetInnerHTML={{ __html: introText }} />
+              )}
+            </Box>
+          </Box>
+        </TextContainer>
+
+        {chapters.map((chapter, i) => {
+          const isDark = i % 2 === 0
+          return (
+            <ChapterWrapper isDark={isDark} key={i}>
```

Comment:
How about using the `<Box>` component instead of defining a `<ChapterWrapper>`?

```tsx
<Box {...(isDark ? { color: "white", backgroundColor: "black100" } : {})}>
</Box>
```

Reply:
A good suggestion but ended up needing another style declaration, so keeping for now.